awscrt/s3.py

```diff
@@ -586,3 +586,18 @@ def is_optimized_for_system():
             for the current system.
     """
     return _awscrt.s3_is_crt_s3_optimized_for_system()
+
+
+def get_recommended_max_throughput_gbps():
+    """
+        Returns:
+        Recommended max throughput in Gbps for the CRT S3 client based on detected system configuration as a PyFloat.
+        If the best throughput configuration is unknown, returns 5Gbps.
```

Comment:
Spell out gigabits per second in the docs. I've seen plenty of confusion in various codebases on what "Gb" means.

Also, PyFloat is a c-api term. It's just called `float` in python. And you could just omit this if you put type annotation on the function `-> float:` and/or said the default was `5.0`
```suggestion
    Returns:
        Recommended throughput, in gigabits per second, based on detected system configuration.
        If the best throughput configuration is unknown, returns 5.0 Gbps.
        Use this as the S3Client's `throughput_target_gbps`.
``````